plugins/videobasedtracker/com_osvr_VideoBasedHMDTracker.cpp

```diff
@@ -539,11 +574,63 @@ class ConfiguredDeviceConstructor {
     // Using `get` here instead of `[]` lets us provide a default value.
     int cameraID = root.get("cameraID", 0).asInt();
     bool showDebug = root.get("showDebug", false).asBool();
+    size_t solveIterations = root.get("solveIterations", 0).asInt();
+    double maxReprojectionAxisError = root.get("maxReprojectionAxisError", 0).asDouble();
+
+    // Default is an empty list, which can be filled in from the config
+    // file if we find entries there.
+    osvr::vbtracker::SensorDescriptionList mySensors;
+    if (root.isMember("sensors")) {
+      Json::Value sensors = root["sensors"];
+      for (Json::ArrayIndex i = 0; i < sensors.size(); ++i) {
```

Comment:
can actually use a range-based for-loop here: `for (auto const & sensor : sensors)` and simplify the subsequent code